packages/components/src/components/event/eventContent/EventContent.tsx

```diff
@@ -90,7 +114,18 @@ const EventContent: React.FC<Props> = ({
             {!isInternetEvent && <EventLocation event={event} />}
           </div>
           <div>
-            <EventInfo event={event} superEvent={superEvent} />
+            <EventInfo
+              appConfig={AppConfig}
+              event={event}
+              superEvent={superEvent}
+              getPlainEventUrl={getPlainEventUrl}
+              iconDirections={iconDirections}
+              getOrganizationSearchUrl={getOrganizationSearchUrl}
+              useSubEvent={useSubEvent}
+              useSubEventsQueryVariables={useSubEventsQueryVariables}
+              useOtherEventTimes={useOtherEventTimes}
```

Comment:
The hooks are prop-drilled here, which is not a good thing either. It would be easier to understand, to test and to handle if the related setters and getters or anything that the hook offers would have been used instead. Or if the hook is always the same, the whole hook should be moved to the components-package, so it could be used normally, without prop-drilling.